server/src/main/java/org/elasticsearch/bootstrap/JNANatives.java

```diff
@@ -153,19 +153,6 @@ static String rlimitToString(long value) {
         }
     }
 
-    /** Returns true if user is root, false if not, or if we don't know */
-    static boolean definitelyRunningAsRoot() {
-        if (Constants.WINDOWS) {
-            return false; // don't know
-        }
-        try {
-            return JNACLibrary.geteuid() == 0;
```

Comment:
Should we remove `geteuid()` from `JNACLibrary`?

Reply:
Oops, yes, removed.